src/main/java/com/epam/productsreview/service/ReviewServiceRestEurekaClientImpl.java

```
@@ -34,17 +34,22 @@
     private String reviewsEndpoint="/reviews/product/";
     private String reviewEndPt="/reviews/";
     @Autowired
-    EurekaClient eurekaCleint;
-
+    private EurekaClient eurekaCleint;
 
     public ResponseEntity getReviewsByProductId(int productId) {
         List<InstanceInfo> instances =
                 eurekaCleint.getApplication(reviewServiceId).getInstances();
         InstanceInfo instanceInfo = instances.get(0);
         String url = "http://" + instanceInfo.getIPAddr() + ":" + instanceInfo.getPort();
-        logger.info(url);
-        ResponseEntity resentity = restTemplate.exchange(url + reviewsEndpoint + productId,
+        logger.info(">>>>>>>>>>"+url + reviewsEndpoint + productId);
+        
+        ResponseEntity resentity =null;
+        try {
+        	resentity= restTemplate.exchange(url + reviewsEndpoint + productId,
                 HttpMethod.GET, getEntity(), String.class);
+        }catch(Exception ex) {
+        	logger.error("Error in rendering",ex);
```

rethrow exceptions instead of swallowing them